ccd-definition/CaseEvent/AnyState.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "internal-change:SEND-DOCUMENT",
```

Comment:
hmm I am not sure internal-change is right here. I believe we wanted to use this phrasing for when there is an internal change of state. I would suggest something like internal-event? Thoughts?